### PR TITLE
Add OAuth error page

### PR DESCRIPTION
**Pull request by cameronmarsh**

### CODE REVIEW

File: src/pages/oauth-error-view/oauth-error-view.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import routes from "../../routes.js"
+
+const getError = search => {
+  return qs.parse(search, { ignoreQueryPrefix: true }).error_description
+}
+
+const OAuthErrorView = () => {
+  return (
+    <oauth-error-view>
+        <h2>Oops! We can't process that request. Please click <a href={routes.root}>here</a> to go back to the Operating System home page. </h2>
```
Comment (mcsearchin) [inline src/pages/oauth-error-view/oauth-error-view.js]: Do we want to use the phrase "Operating System" here?  Seems Columbus-specific.

Comment (cameronmarsh) [reply src/pages/oauth-error-view/oauth-error-view.js]: ya ya ya

File: src/pages/oauth-error-view/oauth-error-view.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom'
+import './oauth-error-view.scss'
+import qs from 'qs'
+import routes from "../../routes.js"
+
+const getError = search => {
+  return qs.parse(search, { ignoreQueryPrefix: true }).error_description
```
Comment (mcsearchin) [inline src/pages/oauth-error-view/oauth-error-view.js]: Is `error_description` a parameter that auth0 will provide when redirecting to this page? 
Also, does all of the config for going to this page exist on the auth0 server side?  No config that needs to happen here in the front end?

Comment (smillardHNTB) [reply src/pages/oauth-error-view/oauth-error-view.js]: Yes - auth0 populates it. 
We will need to modify the auth0 config - going to do that in auth0-setup next. There is no config in the front end needed.

**Pull request merged**